kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java

```diff
@@ -262,7 +264,18 @@ public static List<String> getLanguagesOfAttachments(ResourceReference resourceR
     return languages;
   }
 
-  private void init() {
+  public boolean isPasteNodeAllowed() throws ClipboardException {
+    SilverLogger.getLogger(this).info("isPasteNodeAllowed() nbSelectedObjects = {0}", getClipboardSelectedObjects().size());
+    return getClipboardSelectedObjects().stream().anyMatch(s->s.isDataFlavorSupported(NodeSelection.NodeDetailFlavor));
+  }
+
+  public boolean isPastePublicationAllowed(boolean isRoot) throws ClipboardException {
+    SilverLogger.getLogger(this).info("isPastePublicationAllowed() isRoot = {0} - nbSelectedObjects = {1}", isRoot,getClipboardSelectedObjects().size());
```

Comment:
This is more a debug info than a productive information here. So the `debug(...)` method should be used instead of the `info(...)` one.